client/payment-methods-map.js

```diff
@@ -205,4 +205,15 @@ export default {
 		Icon: icons.multibanco,
 		currencies: [ 'EUR' ],
 	},
+	cashapp: {
+		id: 'cashapp',
+		label: __( 'Cash App Pay', 'woocommerce-gateway-stripe' ),
+		description: __(
+			'Cash App is a popular consumer app in the US that allows customers to bank, invest, send, and receive money using their digital wallet.',
```

Comment:
This sentence comes directly from Stripe's doc here: https://docs.stripe.com/payments/cash-app-pay. I'm not sure if we have a specific string that should go here.